modules/installation-localzone-generate-k8s-manifest.adoc

```diff
@@ -26,8 +26,57 @@ $ ./openshift-install create manifests --dir <installation_directory> <1>
 <1> For `<installation_directory>`, specify the installation directory that
 contains the `install-config.yaml` file you created.
 
-. Create the machine set manifests for the worker nodes in your Local Zone.
+. Set the default Maximum Transmission Unit (MTU) according to the network plugin:
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation. 
+The Cluster network MTU must be always less than the EC2 MTU to account for the cluster network overlay overhead. The specific overhead is determined by your network plugin:
+
+- OVN-Kubernetes: 100 bytes
```

Comment:
OVN-IPsec overhead is an extra 46 bytes, this is be documented in the IPsec page `modules/nw-ovn-ipsec-enable.adoc`

I assume nobody is using IPsec with Local Zones.

Reply:
I avoid covering all variants here. I am assuming the user is being notified and should check the additional resources for more information. If you think that it is required to add here, please let me know.